databases/week1/week1-homework.sql

```diff
@@ -0,0 +1,63 @@
+-- https://github.com/HackYourFuture-CPH/databases/blob/main/lesson1/lesson-data.sql
+
+
+-- Find out how many tasks are in the task table
+SELECT COUNT(*) AS task_count 
+FROM task;
+
+
+-- Find out how many tasks in the task table do not have a valid due date
+SELECT COUNT(*) AS task_count_no_due_date 
+FROM task 
+WHERE due_date IS NULL;
+
+
+-- Find all the tasks that are marked as done
+SELECT *
+FROM task
+WHERE task.status_id = 3;
+
```

Comment:
The assignment is to compare the status to 'done', not using the status_id. For this, you need to do a JOIN.